green/test-of-container-images-regarding-known-vulnerabilities.md

```diff
@@ -0,0 +1,23 @@
+# Test of Container Images Regarding Known Vulnerabilities
+
+Perform tests for known vulnerabilities in all container images used by your software.
+
+Note, that this can result in many found vulnerabilities. Check, whether they are fixed with the next build of the container image. See related Belt Activity: Container Update Enforcement.
```

Comment:
```suggestion
Note that this can result in many found vulnerabilities. Check whether they are fixed in the next build of the container image. See related Belt Activity: Container Update Enforcement.
```